### ISSUE TITLE
MSN avatar didn't get changed when updating globally

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dequis/msn-pecan#46]
```
I'm not sure whether this is a bug in Pidgin or just in pecan, but I see it
in pecan so I'll file it here.

What steps will reproduce the problem?
1. Change avatar in bottom right of Pidgin main window 

What is the expected output? What do you see instead?

I expect that my MSN contacts will see the new icon just by my doing this.
But I had to go to Accounts->WLM->Edit Account and change it there as well.

What version of the product are you using? On what operating system?
0.12 on Hardy from the .deb

```

Original issue reported on code.google.com by `giraffeh...@gmail.com` on 25 May 2008 at 10:40

**Comment by GoogleCodeExporter** [comment]
```
Changing avatar from the main window using the MSN plugin provided with Pidgin 
works
fine. Must be a Pecan thing then.
```

Original comment by `iai...@gmail.com` on 25 May 2008 at 10:45

**Comment by GoogleCodeExporter** [comment]
```
I can confirm that this issue is happening for me as well, with Adium version 
1.3b7.
The relevant Adium bug is over here: http://trac.adiumx.com/ticket/10411
```

Original comment by `alan.hus...@gmail.com` on 16 Jul 2008 at 4:10

**Comment by GoogleCodeExporter** [comment]
```
And a related problem: If you change avatars on MSN plugin by pidgin all is 
fine; if
you use those SAME avatars on pecan all is fine. The problem is when you try to 
use
avatars that you haven't used on MSN plugin by pidgin, then those avatars won't 
get
transmited and your avatar stays the same.
```

Original comment by `esteban....@gmail.com` on 6 Aug 2008 at 6:19

**Comment by GoogleCodeExporter** [comment]
```
Also Adium needs an urgent fix for this for 1.3 final version (or this will be a
regression from 1.2.* version).
```

Original comment by `devid...@gmail.com` on 7 Aug 2008 at 11:28
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in git, would be on the next release.
```

Original comment by `felipe.contreras` on 12 Aug 2008 at 11:19
- Changed state: **Fixed**
